hazelcast/src/test/java/com/hazelcast/json/MapAggregationJsonTest.java

```diff
@@ -78,6 +78,7 @@ public void setup() {
         factory = createHazelcastInstanceFactory(3);
         factory.newInstances(getConfig(), 3);
         instance = factory.getAllHazelcastInstances().iterator().next();
+        warmUpPartitions(factory.getAllHazelcastInstances());
```

Comment:
what about creating this 3 node cluster once in a test suite and updating test methods accordingly?

Reply:
For that, I need to parameterize the static `@BeforeClass` method, since `getConfig` is called with different parameters (`inMemoryFormat` and `metadataPolicy`). Is there a way to do that?

Reply:
needs some polishing but this can be an idea to try: https://gist.github.com/ahmetmircik/844ddcade1aea87a8a51bee01a9cbdb2

Reply:
Thanks for the idea Ahmet, updated the tests accordingly. Also, changed the `MapIndexJsonTest` with the same approach. I will prepare the EE PR soon